pkg/vetter/serviceportprefix/vet.go

```diff
@@ -30,8 +32,10 @@ const (
 	servicePortPrefixNoteType = "missing-service-port-prefix"
 	servicePortPrefixSummary  = "Missing prefix in service - ${service_name}"
 	servicePortPrefixMsg      = "The service ${service_name} in namespace ${namespace}" +
-		" contains port names not prefixed with mesh supported protocols." +
-		" Consider updating the service port name with one of the mesh recognized prefixes."
+		" contains the following port name(s) not prefixed with mesh supported" +
+		" protocols: ${port_prefixes}." +
+		" Consider updating the service port name with one of the mesh recognized prefixes." +
```

Comment:
Update the vetter docs https://github.com/aspenmesh/istio-vet/tree/master/pkg/vetter/serviceportprefix#service-port-prefix to include the new list and refer to the vetter docs instead as the istio docs might get updated which will break the vetter.